plugins/php/php_fpm_status

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# 	Plugin to monitor php-fpm process manager, php-fpm 5.3.3 is required
+#   Copyright (C) 2010 radar AT aol DOT pl
+#	Copyright (C) 2011 Daniel Caillibaud
+#	Copyright (C) 2020 Rowan Wookey <https://www.rwky.net>
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# 
+#  # INSTALLATION
+#  1. Install curl or wget
+#  2. Configure PHP to respond to status request by setting pm.status_path
+#  in your php fpm config file. 
+#  3. Configure your websever to respond to this path, it's advisable to restrict
+#  this to localhost
+#  4. Configure munin-node with a url environment variable for the url you wish to test
+# 
+#  You can use symlinks and config files to monitor multiple pools
+#  ln -s /usr/share/munin/php_fpm_status /usr/share/munin/php_fpm_status_www
+#  ln -s /usr/share/munin/php_fpm_status /usr/share/munin/php_fpm_status_www2
+# 
+#  [php_fpm_status_www]
+#  env.url http://127.0.0.1/fpm-status-www
+#  [php_fpm_status_www2]
+#  env.url http://127.0.0.1/fpm-status-www2
+# 
+#%# family=contrib
+
+url=${url:-'https://127.0.0.1/fpm-status'}
+
+if [ -z "$url" ]
+then
+  echo "env.url not specified" >&2
+  exit 1
+fi
+if which wget > /dev/null 2>&1
+then
+  wget -q -O "$MUNIN_STATEFILE" "$url"
```

Comment:
You could easily avoid using a temporary file here, by assigning the output to a variable (`foo=$(wget -O - ...)`).